ddtrace/_trace/trace_handlers.py

```diff
@@ -691,6 +691,28 @@ def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_tok
     span.set_tag_str("bedrock.usage.completion_tokens", output_token_count)
 
 
+def _on_rq_queue_enqueue_job_is_async(ctx, job):
```

Comment:
Can we make the function name more generic so we can reuse this function in other integrations.

```suggestion
def _on_propagate_context(ctx, job):
```